docs/index.md

```diff
@@ -176,31 +176,31 @@ It is generally recommended to consider the [visibility range](http://doc.arcgis
 
 You'll want to consider whether to enable or disable [editing](http://doc.arcgis.com/en/arcgis-online/manage-data/edit-features.htm) of your feature layers and tables. Specifically, a user is only able to edit features or records on layers whose backing table has editing enabled. This includes related records for features. For instance, if a feature whose backing table does permit editing has a related record backed by a table that does not have editing enabled, that related record layer cannot be edited by the app.
 
-#### Enable popup on feature layers and tables
+#### Enable pop-up on feature layers and tables
 
-The app relies on popup configurations to identify, view, and edit features and records. You'll want to consider whether to enable or disable [popups](https://doc.arcgis.com/en/arcgis-online/create-maps/configure-popups.htm#ESRI_SECTION1_9E13E02AABA74D5DA2DF1A34F7FB3C63) of your feature layers and tables. Only feature layers and tables that are popup-enabled can be identified or edited. Please note, you can have a scenario where you've enabled editing on a layer (as described above) but have disabled popups for the same layer and thus a user will not be able to edit this layer.
+The app relies on pop-up configurations to identify, view, and edit features and records. You'll want to consider whether to enable or disable [pop-ups](https://doc.arcgis.com/en/arcgis-online/create-maps/configure-popups.htm#ESRI_SECTION1_9E13E02AABA74D5DA2DF1A34F7FB3C63) of your feature layers and tables. Only feature layers and tables that are pop-up-enabled can be identified or edited. Please note, you can have a scenario where you've enabled editing on a layer (as described above) but have disabled pop-ups for the same layer and thus a user will not be able to edit this layer.
```

Comment:
This link broke when I tried to explore to it. It just needs a hyphen in `popups`

```diff
- https://doc.arcgis.com/en/arcgis-online/create-maps/configure-popups.htm#ESRI_SECTION1_9E13E02AABA74D5DA2DF1A34F7FB3C63
+ https://doc.arcgis.com/en/arcgis-online/create-maps/configure-pop-ups.htm#ESRI_SECTION1_9E13E02AABA74D5DA2DF1A34F7FB3C63
```